feat(metric): add metric query (#160)
propeldata/ui-kit: .changeset/fifty-zebras-check.md

---
'@propeldata/ui-kit': minor
---

> feat(metric): add ability to query metrics on-the-fly

Reviewer: Can we remove this prefix? The rest of the changesets don't have it, just to keep consistency
```suggestion
Added ability to query metrics on-the-fly
```

> feat(metric): add metric query for counter, leaderboard and timeseries hooks.

Reviewer 2: Should it be this?

```suggestion
feat(metric): add ability to query metrics on-the-fly
```
